vscode-extensions.json

```diff
@@ -344,7 +344,7 @@
     },
     {
       "repository": "https://github.com/scalameta/metals-vscode",
-      "revision": "v1.9.4",
+      "revision": "v1.9.5",
       "sidecar": {
         "image": "quay.io/eclipse/che-sidecar-scala:0.9.0-b92a5f9",
```

Comment:
here you will need to use the new ID after https://github.com/eclipse/che-plugin-registry/pull/682/files is merged @sideeffffect